signer/tests/integration/postgres.rs

```diff
@@ -294,7 +308,15 @@ async fn should_return_the_same_pending_deposit_requests_as_in_memory_store() {
         num_withdraw_requests_per_block: 5,
         num_signers_per_request: 0,
     };
-    let test_data = testing::storage::model::TestData::generate(&mut rng, &test_model_params);
+    let signer_info = testing::wsts::generate_signer_info(&mut rng, num_signers);
+    let coordinator_signer_info = signer_info.first().cloned().unwrap();
+    let signer_set: Vec<_> = coordinator_signer_info
```

Comment:
You're doing this quite a few times so I'd probably make some sort of helper

Reply:
yeah good point. Will update.

Reply:
https://github.com/stacks-network/sbtc/pull/529/commits/a7bbb2f911139beb51bc022a829dd6fce6a6d20c